apps/docs/guides/jssg/quickstart.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: "Quickstart"
+---
+
+Build a working JavaScript ast-grep (jssg) codemod in minutes.
+
+## What is jssg?
+
+JavaScript ast-grep (jssg) is a codemod framework that lets you write code transformations using JavaScript/TypeScript. Instead of complex AST manipulation, you use patterns to find and replace code structures.
```

Comment:
I find the last sentence a bit confusing. It's as if TOS OS not ast based whereas it actually is